.gitignore

```diff
@@ -20,4 +20,5 @@ server*.cfg
 _savebackup/
 *.swatches
 /imgui.ini
+o3de-packages/
```

Comment:
please revert this change, it may be something you need locally, but is not a common location for the o3de-packages for every other o3de user

```suggestion
```

Reply:
Removed.